Library/Homebrew/diagnostic.rb

```diff
@@ -571,17 +517,18 @@ def check_for_config_scripts
         EOS
       end
 
-      def check_dyld_vars
-        dyld_vars = ENV.keys.grep(/^DYLD_/)
-        return if dyld_vars.empty?
+      def check_ld_vars
+        ld_vars = ENV.keys.grep(/^(|DY)LD_/)
+        return if ld_vars.empty?
 
-        values = dyld_vars.map { |var| "#{var}: #{ENV.fetch(var)}" }
+        values = ld_vars.map { |var| "#{var}: #{ENV.fetch(var)}" }
+        vars = ld_vars.map { |x| x.sub(/_.*/, "") }.uniq.sort.map { |x| x + "_*" }.join(" and ")
         message = inject_file_list values, <<~EOS
-          Setting DYLD_* vars can break dynamic linking.
+          Setting #{vars} variables can break dynamic linking.
```

Comment:
Let's just do `DYLD_* or LD_*` variables and kill the `vars` line above.

Reply:
ok